docs/intro.txt

```diff
@@ -11,6 +11,9 @@ and Windows. To install, run:
 
     pip install pytket-quantinuum
 
+
+.. note:: Users of the Quantinuum Nexus platform do not need to use pytket-quantinuum. See the `code examples <https://docs.quantinuum.com/nexus/examples/overview>`_ for documentation on how to access H-Series with Quantinuum Nexus. 
```

Comment:
Suggested rephrasing:

> `pytket-quantinuum` is not compatible with Quantinuum Nexus. To use your Nexus account please see `the Nexus documentation <https://docs.quantinuum.com/nexus>`_ and the `qnexus <https://pypi.org/project/qnexus/>`_ Python package.

Reply:
I know I asked if we could use the https://docs.quantinuum.com/nexus/examples/overview URL but I think it might be simpler to just send them to the Nexus docs landing page. Thanks!

Reply:
revised wording in [bbf1c80](https://github.com/CQCL/pytket-quantinuum/pull/490/commits/bbf1c801530596163ccfb7f5f4bdf664ccb814e5)